constructor.go

```diff
@@ -135,7 +136,7 @@ func (n *constructorNode) Call(c containerStore) error {
 		}
 	}
 
-	args, err := n.paramList.BuildList(c)
+	args, err := n.paramList.BuildList(c, false)
```

Comment:
```suggestion
	args, err := n.paramList.BuildList(c, false /* decorating */)
```
per https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters